tests/test_app.py

```diff
@@ -3,23 +3,62 @@
 
 import pynautobot
 
+from .util import Response
+
 host = "http://localhost:8000"
 
 def_kwargs = {
     "token": "abc123",
 }
 
 
-class AppCustomChoicesTestCase(unittest.TestCase):
+class AppCustomFieldsTestCase(unittest.TestCase):
     @patch(
-        "pynautobot.core.query.Request.get",
-        return_value={"Testfield1": {"TF1_1": 1, "TF1_2": 2}, "Testfield2": {"TF2_1": 3, "TF2_2": 4}},
+        "requests.sessions.Session.get",
+        side_effect=[Response(fixture="extras/custom_fields.json")],
```

Comment:
```suggestion
        return_value=Response(fixture="extras/custom_fields.json"),
```
`return_value` is good enough, side_effect is for rising exceptions or iterables.